spec/factories/channel/channel_whatsapp.rb

```diff
@@ -34,6 +34,15 @@
     end
     message_templates_last_updated { Time.now.utc }
 
+    transient do
```

Comment:
Why do we need `transient` here?

Reply:
`sync_templates` is called on on_create and makes a network call, which we don't want to happen in some tests.
this transient property here helps us to selectively disable this network call by passing false to this property while creating the `whatsapp_channel` factories. 

```
# sync_method is overridden in this case the transient property is set. 
let(:channel_whatsapp) { create(:channel_whatsapp, sync_templates: false) }
```

ref: https://devhints.io/factory_bot